apps/i18n/common/en_us.json

```diff
@@ -1709,7 +1709,9 @@
   "teacherWithColon": "Teacher:",
   "teachers": "Teachers",
   "teacherAnnouncementSpecial2020Body": "Students can continue to learn at home while schools are closed. Take a Code Break with us, or see resources for students, parents, and teachers - including videos, fun tutorials, and projects!",
+  "teacherAnnouncementSpecialFall2020Body": "See recommended adjustments to Code.org's courses and get answers to frequently-asked-questions about how to run a virtual or socially-distanced classroom with Code.org materials.",
```

Comment:
Should it be "frequently-asked-questions" as here, or "frequently asked questions" as above?

Reply:
ooh I think no hyphens, thank for catching this!